Sources/TuistCore/Models/Package.swift

```diff
@@ -1,34 +1,7 @@
 import Foundation
 import TSCBasic
-import XcodeProj
 
 public enum Package: Equatable {
     case remote(url: String, requirement: Requirement)
     case local(path: AbsolutePath)
-
-    public enum Requirement: Equatable {
```

Comment:
To keep `Package.Requirement` you can consider creating a `typealias`

 ```swift
     public typealias Requirement = TuistCore.Requirement 
```

Reply:
After renaming `Requirement` to `PackageRequirement` I think this won't add much more value. Agree?

Reply:
Requirement is valid for both Packages and Carthage Dependencies - its up to you about how you want to manage that but I would guess you want both to avoid having to use PackageRequirement in CarthageDependency

Reply:
Ok I get your point now. Thanks for the double-check 👍

Reply:
Agree. Thanks for the double-check.

However, if I do the typealias I get:

`Invalid redeclaration of 'Requirement'`

I think that since `Requirement` will be at top level it's enough.